### PR TITLE
Playable Laetitia!

### PR DESCRIPTION
**Pull request by EnderMage99**
## About The Pull Request

This PR gives Laetitia 2 new abilities and makes her appear in LCL! First, She is able to make a gift which can be opened by humans and explodes when opened. She choses if the explosion heals or damages.

Also, She is able to summon her spider friends which disappear 30 seconds after summoning. This is so she is able to defend her self since her melee sucks quite a lot.

## Why It's Good For The Game

More mobs for LCL!

## Changelog
:cl:
add: 2 new abilities for laetitia
add: laetitia now appers in LCL
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm [hunk 1]
```diff
@@ -26,6 +31,91 @@
...
+	icon = 'ModularTegustation/Teguicons/tegu_effects.dmi'
+	icon_state = "prank_gift"
+	var/opening = FALSE
+	var/oneuse = TRUE
+	var/basepower = 25
+	var/strength = 1
+
+/obj/item/laetitia_gift/attack_self(mob/user)
+	if(opening)
+		to_chat(user, "<span class='warning'>You're already opening this gift!</span>")
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm:102]:
```suggestion
		to_chat(user, span_warning("You're already opening this gift!"))
```

**Comment by InsightfulParasite** [comment]
Assigning this to Kirie since i dont know anything about LCL

**Review by Kitsunemitsu — APPROVED** [review]
She's a bit of a shitter, but this is fine for our HRP gamemode.

**Review by Kitsunemitsu — APPROVED** [review]
Re-approving

**Pull request merged**